src/popup/components/Main.vue

```diff
@@ -96,7 +99,7 @@
       </div>
       <hr />
 
-      <div v-if="aliasArray.length > 0">
+      <div>
```

Comment:
ah why do we need to the `if` here? It's to show the existing alias section only when user has at least an alias